common/text-level-modifiers/text-level-modifiers-en.html

```diff
@@ -0,0 +1,147 @@
+---
+{
+	"title": "Text-Level Modifiers",
+	"language":"en",
+	"altLangPage":"text-level-modifiers-fr.html",
+	"breadcrumbs":
+		[
+			{
+				"title": "GCWeb home",
+				"link": "https://wet-boew.github.io/themes-dist/GCWeb/index-en.html"
+			}
+		],
+	"secondlevel": false,
+	"dateModified": "2024-03-06",
+	"share": "true"
+}
+---
+
+<p>In the realm of GCweb, text-level modifiers play a pivotal role in shaping the visual and
+	functional aspects of website text.They enable web developers to fine-tune text attributes such as size, color, weight, and
+	style, ensuring that the textual content is not only aesthetically pleasing but also accessible and easy to read.</p>
+<h2>Overview</h2>
+<dl class="dl-horizontal brdr-bttm">
+	<dt><code>.stretched-link</code></dt>
+	<dd>Works in conjunction with position-relative to set a link's clickable area</dd>
+	<dt><code>.small</code></dt>
+	<dd>Decreases the font size of the element</dd>
+	<dt><code>.mark</code></dt>
+	<dd>Highlights the text</dd>
+	<dt><code>.lead</code></dt>
+	<dd>Increases the font size and line height</dd>
+	<dt><code>.text-*</code></dt>
+	<dd>Changes the alignment or case of the text</dd>
+	<dt><code>.no-wrap</code></dt>
+	<dd>Prevents the text from wrapping</dd>
+	<dt><code>.fnt-nrml</code></dt>
+	<dd>Resets the font size to the default</dd>
+</dl>
+<div class="row">
+	<div class="position-relative">
+		<h2><code>.stretched-link</code></h2>
+		<dd>Used to make an entire containing block clickable, effectively "stretching" a hyperlink over the entire block.
+			Make sure the element that you want to be clickable is positioned. This can be done by setting its CSS position to relative.</dd>
+
+		<a class="stretched-link" href="">Stretched link</a>
+		<pre><code>&lt;div <strong>class=&quot;positon-relative&quot;</strong>&gt;&lt;a <strong>class=&quot;stretched-link&quot; href=&quot;&quot;</strong>&gt;Stretched link&lt;/a&gt;&lt;/div&gt;</code></pre>
+	</div>
+	<div>
+		<h2><code>.small</code></h2>
+		<dd>Decreases the font size of the element.</dd>
+		<p class="small">
+			Example small text
+		</p>
+		<pre><code>&lt;p <strong>class=&quot;small&quot;</strong>&gt;...&lt;/p&gt;</code></pre>
+	</div>
+	<div>
+		<h2><code>.mark</code></h2>
+		<dd>Highlights the text</dd>
+		<p class="mark">
+			Example highlighted text
+		</p>
+		<pre><code>&lt;p <strong>class=&quot;mark&quot;</strong>&gt;...&lt;/p&gt;</code></pre>
+	</div>
+	<div>
+		<h2><code>.lead</code></h2>
+		<dd>Increases the font size and line height</dd>
+		<p class="lead">
+			Example lead text
+		</p>
+		<pre><code>&lt;p <strong>class=&quot;lead&quot;</strong>&gt;...&lt;/p&gt;</code></pre>
+	</div>
+	<div>
+		<h2><code>.text-*</code></h2>
+		<dd>Can be used to align text or change its case.</dd>
+		<h2>Alignment examples:</h2>
```

Comment:
This is a sub-heading of the "h2" at line 73. 

```suggestion
		<h3>Alignment examples:</h3>
```

And apply the similar fix for the other similar sub-heading